packages/legend-studio/src/components/editor/edit-panel/element-generation-editor/FileGenerationEditor.tsx

```diff
@@ -217,6 +218,32 @@ export const GenerationResultExplorer = observer(
   },
 );
 
+export const extraFileGenerationResultViewerActionsGetter = (
```

Comment:
Don't think you need this, you can just add a `map` directly after getting the plugins and call the creator function to generate the list of buttons to render directly.

Reply:
Fixed -- using map inside and removed redundant getter.